addons/website_slides/static/src/js/slides_course_enroll_email.js

```diff
@@ -58,7 +58,7 @@ publicWidget.registry.websiteSlidesEnroll = publicWidget.Widget.extend({
     _openDialog: function (channelId) {
         new SlideEnrollDialog(this, {
             channelId: channelId,
-            $element: this.$el
+            $element: this.$el.closest('.alert')
```

Comment:
Mmmh, I don't really see how this is related ^^

Reply:
I'm guessing you re-tested the other use case(s) then? :slightly_smiling_face:

Reply:
Yes I did :wink: